server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/model/MultipleDatabaseChangeParameters.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.flow.task.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import com.oceanbase.odc.core.flow.model.TaskParameters;
+import com.oceanbase.odc.core.shared.constant.TaskErrorStrategy;
+import com.oceanbase.odc.service.connection.database.model.Database;
+import com.oceanbase.odc.service.schedule.model.JobType;
+
+import lombok.Data;
+
+/**
+ * @author: zijia.cj
+ * @date: 2024/3/27
+ */
+@Data
+public class MultipleDatabaseChangeParameters implements Serializable, TaskParameters {
```

Comment:
may resuse model DatabaseChangeParameters.

extends it, or refer it

MultipleDatabaseChangeParameters
- DatabaseChangeParameters 
- orderedDatabaseIds

Reply:
This is returned to the front end as a model, and the front end renders with this structure, so extends it is appropriate